.github/workflows/repeat_build.yml

```diff
@@ -27,9 +27,10 @@ on:
         options:
           - publish
           - draft
+          - build
```

Comment:
Ooh question on draft - would we want to supply the full string for the draft revision here? As in `24v1/1`? I think that's probably simplest. Sorry if I just haven't gotten to where you deal with this

Reply:
I think it might be annoying for consecutive drafts... Say something like `24v1/3-fix-zoning-data-and-update-sql-logic`. It shouldn't be too bad to have a separate logic for drafts: select version and then select draft